src/parser/bison_parser.y

```diff
@@ -374,24 +383,65 @@ execute_statement:
 
 /******************************
  * Import Statement
+ * IMPORT FROM TBL FILE 'test/students.tbl' INTO students
+ * COPY students FROM 'test/students.tbl' (WITH FORMAT TBL)
  ******************************/
 import_statement:
 		IMPORT FROM import_file_type FILE file_path INTO table_name {
-			$$ = new ImportStatement((ImportType) $3);
+			$$ = new ImportStatement($3);
 			$$->filePath = $5;
 			$$->schema = $7.schema;
 			$$->tableName = $7.name;
 		}
+	|	COPY table_name FROM file_path opt_file_type {
+			$$ = new ImportStatement($5);
+			$$->filePath = $4;
+			$$->schema = $2.schema;
+			$$->tableName = $2.name;
+		}
 	;
 
 import_file_type:
-		CSV { $$ = kImportCSV; }
+		IDENTIFIER {
+			if (strcasecmp($1, "csv") == 0) {
+			 	$$ = kImportCSV;
+			} else if (strcasecmp($1, "tbl") == 0) {
+			 	$$ = kImportTbl;
+			} else if (strcasecmp($1, "binary") == 0) {
+			 	$$ = kImportBinary;
+			} else {
+				free($1);
+				yyerror(&yyloc, result, scanner, "File type is unknown.");
+			 	YYERROR;
+			}
+			free($1);
+		}
 	;
 
 file_path:
 		string_literal { $$ = strdup($1->name); delete $1; }
 	;
 
+opt_file_type:
+		WITH FORMAT import_file_type {
+			$$ = $3;
+		}
+	|	/* empty */  { $$ = kImportAuto; }
+	;
+
+
+/******************************
+ * Export Statement
+ * COPY students TO 'test/students.tbl' (WITH FORMAT TBL)
+ ******************************/
+export_statement:
+		COPY table_name TO file_path opt_file_type {
```

Comment:
As this uses opt_file_type, too, it is weird that opt_file_type has a import_file_type in an export context